doc/manual/syntax.md

```diff
@@ -0,0 +1,221 @@
+| \-       | The subtraction operator                             | `1 - 2 = -1`  |
+| *        | The multiplication operator                          | `1 * 2 = 2`   |
+| /        | The division operator                                | `1 / 2 = 0.5` |
+| %        | The modulo operator (returns the *signed* remainder) | `5 % 3 = 2`   |
+
+Numbers are also comparable, using the dedicated comparison operators, returning a boolean value :
+| Operator | Description      | Example   |
+|:--------:|:----------------:|:---------:|
+| ==       | Equal            | `5 == 5`  |
+| !=       | Not Equal        | `5 != 4`  |
+| <        | Smaller than     | `2 < 3`   |
+| >        | Greater than     | `1 > -5`  |
+| >=       | Greater or Equal | `1 >= 1`  |
+| <=       | Smaller or Equal | `-1 <= 6` |
+
+
+### Boolean values
+
+The boolean values in Nickel are denoted `true` and `false`.
+
+To work with boolean values, we have a the classic logical operators *AND* (&&), *OR* (||) and *NOT* (!).
+The _AND_ and _OR_ operators are lazy in the evaluation of the second argument. The truth table below gives the results for using these operators:
+
+Examples:
+```
+> true && false
+false
+
+> false || true
+true
+
+> ! true
+false
+```
+
+### Strings
+
+Nickel can work with sequences of characters, or strings. 
+Strings are enclosed by `" ... "` for a single line string or by `m#" ... "#m` for a multiline string.
+They can be concatenated with the operator `++`.
+Strings must be UTF-8 valid.
+
+Examples:
+```
+> "Hello, World!"
+"Hello, World!"
+
+> m#"Well, if this isn't a multiline string?
+Yes it is, indeed it is"#m
+"Well, if this isn't a string?
+Yes it is, indeed it is"
+
+> "Hello" ++ "World"
+"HelloWorld"
+```
+
+## Equality Operators
+
+We have seen that the operators `==` and `!=` works for comparing numbers, but they can actually compare any values together. Two values are equal if they have the same _type_ and the same _value_.
+
+Examples:
+```
+> 1 == 1
+true
+
+> 5 == 5.0
+true
+
+> "Hello" == "Hello"
+true
+
+> "Hello" != "World"
+true
+
+> 5 == "Hello"
+false
+
+> true == "true"
+false
+```
+
+
+## Composite types
+
+### List
+Lists allows grouping of values into sequences. They are delimited by `[` and `]`, and elements are separated with `,`.
+
+Examples:
+```
+[1, 2, 3]
+["Hello", "World"]
+[1, true, "true"]
+[]
+```
+
+Lists can be concatenated with the operator `@`:
+```
+> [1] @ [2, 3]
+[ 1, 2, 3 ]
+```
+
+### Record
+Records are key-value storage. They are delimited with by `{` and `}`, elements are separated by `,`.
+key-value elements are noted as `key = value`.
+The keys can be either identifiers, or strings. Values can be of any type.
+Elements inside a record are unordered.
+
+Examples:
+```
+{}
+{a = 3}
+{my_id_n5 = "my id number 5", my_id_n4 = }
+{"5" = 5, six = 6}
+```
+
+A record cannot contain the same key twice, or the values must be identical (see section about _merge_).
```

Comment:
That's true for primitive values, but not for records. For example `{foo = {bar = 1}, foo = {baz = 2}}` is totally fine, and is what `{foo.bar = 1, foo.baz = 2}` is desugared to. You can either say that exactly (primitive values must be the same, but records are treated differently) or totally offload the discussion to the merging section, just saying that multiple definitions are merged.

Reply:
removed the section, the reader should read the documentation about merge.